mkdocs/themes/mkdocs/locales/ru/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,101 @@
+# German translations for MkDocs.
+# Copyright (C) 2022 MkDocs
+# This file is distributed under the same license as the MkDocs project.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: MkDocs 1.2\n"
+"Report-Msgid-Bugs-To: https://github.com/mkdocs/mkdocs/issues\n"
+"POT-Creation-Date: 2022-09-13 23:23+0300\n"
+"PO-Revision-Date: 2022-09-13 23:23+0300\n"
+"Last-Translator: Alexander Krasnikov <alexander@krasnikov.net>\n"
+"Language: ru\n"
+"Language-Team: ru <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Ah and I see also that plurals are wrongly declared because you didn't run `python setup.py init_catalog -t mkdocs -l ru`

```suggestion
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
```